app/views/courses/salary.html.erb

```diff
@@ -0,0 +1,68 @@
+<%= content_for :page_title, "#{course.name_and_code} - Course length and salary" %>
```

Comment:
Can we front-load the title with the most relevant bit?

Reply:
Done